### PR TITLE
Changed typos from 'plomber' to 'ploomber'.

### PR DESCRIPTION
**Pull request by RobsonGlasscock**
Files modified were: setup.py, cloud.py, nb.py, and notebooksource.py.

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -89,7 +89,7 @@ def read(name):
     # needed to run some test pipelines
     'matplotlib',
     'seaborn',
-    'psycopg2',
+    #'psycopg2',
```
Comment (idomic) [inline setup.py:92]: Please make sure it's enabled.

**Review by idomic — CHANGES_REQUESTED** [review]
Looks good besides the psycopg2 change, please fix

**Comment by RobsonGlasscock** [comment]
Thank you for the feedback. The change has been made.

**Comment by idomic** [comment]
@RobsonGlasscock please make sure the CI is passing, (rebasing on the master should solve this)

**Pull request closed without merging**